core/sql/sqlcomp/nadefaults.cpp

```diff
@@ -1958,6 +1958,7 @@ SDDkwd__(EXE_DIAGNOSTIC_EVENTS,		"OFF"),
 
   DDkwd__(HIVE_DEFAULT_CHARSET,            (char *)SQLCHARSETSTRING_UTF8),
   DD_____(HIVE_DEFAULT_SCHEMA,                  "HIVE"),
+  DD_____(HIVE_FILE_CHARSET,            (char *)SQLCHARSETSTRING_UTF8),
```

Comment:
Yes, but that's also a problem with the current implementation, with fully supported charsets. We can't join two tables with ISO and UTF8, either. IMHO the solution to that is called iconv.